lte/gateway/python/magma/pipelined/app/gy.py

```diff
@@ -137,8 +137,9 @@ def _deactivate_flows_for_subscriber(self, imsi, ip_addr):
             self._datapath,
             imsi,
         )
-        self._qos_mgr.remove_subscriber_qos(imsi)
-        self._remove_he_flows(ip_addr, None)
+        if self._qos_mgr:
```

Comment:
Actually this is not possible, this is unconditionally called during initialization. Is there real issue here?

Reply:
Yes, there are the following issues here: https://github.com/magma/magma/issues/10342, https://github.com/magma/magma/issues/10343.

Could you be more specific on what you mean by "this", please?

Reply:
'self._qos_mgr' is initialized at initialize_on_connect(), can you investigate why '_qos_mgr' is None